UnitTests/BraintreePayPalTests/BTPayPalVaultRequest_Tests.swift

```diff
@@ -63,4 +67,16 @@ class BTPayPalVaultRequest_Tests: XCTestCase {
         XCTAssertEqual(shippingParams["country_code"], "US")
         XCTAssertEqual(shippingParams["recipient_name"], "Recipient")
     }
+    
+    func testParameters_withEnablePayPalAppSwitchTrue_returnsAllParams() {
+        BTAppContextSwitcher.sharedInstance.universalLink = "some-url"
```

Comment:
@jaxdesmarais Should we make[ this property](https://github.com/braintree/braintree_ios/blob/f09dfb24be6cc2236ccb1de844dad89786376563/Sources/BraintreeCore/BTAppContextSwitcher.swift#L20) a `URL` type instead?

Reply:
(Outside the scope of this PR, but wanted to ask)

Reply:
🤔 I am down to change it to a URL type instead.

Reply:
Cool, we can do that in another PR. Might as well be more defensive versus just using a `String` type

Reply:
Created DTBTSDK-3659 for this